cir_traj.cpp

```diff
@@ -0,0 +1,19 @@
+#include <ros/ros.h>
+#include <nav_msgs/Odometry.h>
+
+int main(int argc,char** argv){
+    ros::init(argc,argv,"cir_traj");
+    ros::NodeHandle nh;
+    ros::Publisher pub=nh.advertise<nav_msgs::Odometry>("/husky_velocity_controller/odom",10);
```

Comment:
I don't believe this is the topic where you publish the command for pose. This just gives the current odometry of the husky. Try to find the topic that actually commands the husky